### PR TITLE
Switch to Xunit RTM beta and update TestSuite to be a metapackage.

### PR DESCRIPTION
**Pull request by karajas**
Revert xunit.netcore.extensions to netstandard1.1 to remove warnings.

This is in progress work to fix https://github.com/dotnet/corefx/issues/9952 
/cc @weshaggard @Chrisboh @ericstj

### CODE REVIEW

**Comment by karajas** [comment]
@dotnet-bot test Innerloop Windows Debug

File: src/xunit.console.netcore/xunit.console.netcore.csproj [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <NuGetRuntimeIdentifier>win7-x64</NuGetRuntimeIdentifier>
-    <NuGetTargetMoniker>.NETCoreApp,Version=v1.0</NuGetTargetMoniker>
+    <NuGetTargetMoniker>.NETCoreApp,Version=v1.1</NuGetTargetMoniker>
```
Comment (weshaggard) [inline src/xunit.console.netcore/xunit.console.netcore.csproj]: Why does this need to move to netcoreapp1.1?

File: src/nuget/Microsoft.xunit.netcore.extensions.nuspec [hunk 2]
```diff
@@ -14,30 +14,30 @@
...
-        <dependency id="System.Reflection.Primitives" version="4.0.1-rc3-24117-00" />
-        <dependency id="System.Runtime" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Runtime.Extensions" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Runtime.Handles" version="4.0.1-rc3-24117-00" />
-        <dependency id="System.Runtime.InteropServices" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.0.0-rc3-24117-00" />
-        <dependency id="System.Text.Encoding" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.Threading" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.Threading.Tasks" version="4.0.11-rc3-24117-00" />
+      <group targetFramework="netstandard1.0">
```
Comment (weshaggard) [inline src/nuget/Microsoft.xunit.netcore.extensions.nuspec]: Should this be netstandard1.1?

File: src/nuget/xunit.console.netcore.nuspec [hunk 3]
```diff
@@ -18,26 +18,26 @@
...
-        <dependency id="System.IO.FileSystem" version="4.0.1-rc3-24117-00" />
-        <dependency id="System.IO.FileSystem.Primitives" version="4.0.1-rc3-24117-00" />
-        <dependency id="System.Linq" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Runtime" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Runtime.Extensions" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Text.RegularExpressions" version="4.0.12-rc3-24117-00" />
-        <dependency id="System.Threading" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.Threading.Tasks" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.Xml.XDocument" version="4.0.11-rc3-24117-00" />
+      <group targetFramework="netstandard1.0">
```
Comment (weshaggard) [inline src/nuget/xunit.console.netcore.nuspec]: should this be netstandard1.1?

File: src/nuget/Microsoft.xunit.netcore.extensions.nuspec [hunk 4]
```diff
@@ -14,30 +14,30 @@
...
+        <dependency id="System.Diagnostics.Debug" version="4.0.11" />
+        <dependency id="System.IO" version="4.1.0" />
+        <dependency id="System.Linq" version="4.1.0" />
+        <dependency id="System.Reflection" version="4.1.0" />
+        <dependency id="System.Reflection.Primitives" version="4.0.1" />
+        <dependency id="System.Runtime" version="4.1.0" />
+        <dependency id="System.Runtime.Extensions" version="4.1.0" />
+        <dependency id="System.Runtime.Handles" version="4.0.1" />
+        <dependency id="System.Runtime.InteropServices" version="4.1.0" />
+        <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.0.0" />
```
Comment (weshaggard) [inline src/nuget/Microsoft.xunit.netcore.extensions.nuspec]: Correct me if I'm wrong but I thought RuntimeInformation was only supported on netstandard1.1 and higher.

File: src/nuget/xunit.console.netcore.nuspec (hunk 3 above)
Comment (karajas) [inline src/nuget/xunit.console.netcore.nuspec]: I thought this would be netstandard1.0? The xunit.netcore.extensions is the one that requires 1.1.

Comment (weshaggard) [inline src/nuget/xunit.console.netcore.nuspec]: I believe some of these dependent packages don't even support netstandard1.0.

Comment (karajas) [inline src/nuget/xunit.console.netcore.nuspec]: I can bump this to 1.1 but this does seem to be working at the moment.

File: src/nuget/Microsoft.xunit.netcore.extensions.nuspec (hunk 4 above)
Comment (karajas) [inline src/nuget/Microsoft.xunit.netcore.extensions.nuspec]: Yup that's what I was seeing as well

**Comment by karajas** [comment]
Does this look ok at the moment? This greatly a large chunk corefx warnings and I am looking to clean up the remaining

**Comment by weshaggard** [comment]
LGTM. However I'm assuming you have tested it in corefx.

**Comment by karajas** [comment]
Yes at the moment this brings warning count down to 93.

**Comment by markwilkie** [comment]
@crummel who should also know about this

**Pull request merged**